content/en/user-guide/aws/eks/index.md

```diff
@@ -551,6 +549,25 @@ spec:
   restartPolicy: Always
 ```
 
+## Supported Versions
+
+LocalStack uses [k3s](https://github.com/k3s-io/k3s) under the hood for creating EKS clusters.
+Below is the list of supported Kubernetes versions and their corresponding k3s versions.
+The default version is `1.31`.
+
+| Kubernetes Version | k3s Version       | EKS Platform Version |
+|-------------------|-------------------|---------------------|
+| 1.32              | v1.32.1-k3s1     | eks.3              |
+| 1.31              | v1.31.5-k3s1     | eks.19             |
+| 1.30              | v1.30.9-k3s1     | eks.27             |
+| 1.29              | v1.29.13-k3s1    | eks.30             |
+| 1.28              | v1.28.15-k3s1    | eks.36             |
+| 1.27              | v1.27.16-k3s1    | eks.40             |
+| 1.26              | v1.26.15-k3s1    | eks.42             |
+| 1.25              | v1.25.16-k3s4    | eks.42             |
```

Comment:
I think including the EKS platform version here is fine - but we just mock the return value, it does not really mean anything.